server/monitor/subscriptions.go

```diff
@@ -135,9 +135,8 @@ func (m *Monitor) CreateAndSaveChatSubscription(mmSubscription *storemodels.Glob
 		return
 	}
 
-	if m.metrics != nil {
-		m.metrics.ObserveSubscriptionsCount(metrics.SubscriptionConnected)
-	}
+	m.metrics.ObserveSubscriptionsCount(metrics.SubscriptionConnected)
```

Comment:
Why here we are not using the `GetMetrics` trick?

Reply:
In this case, it's just that the `Monitor` object doesn't expose a `GetMetrics`, unlike the other cases where code accesses the metrics through a pointer to the `Plugin` struct.